### PR TITLE
tunnel: add localproxy command

### PR DESCRIPTION
**Pull request by at-wat**
~After #85~ done

### TODO
- [x] add README

### CODE REVIEW

**Comment by at-wat** [comment]
failing on Go 1.12 due to https://github.com/golang/go/issues/30374

**Comment by at-wat** [comment]
@kt3k please take a look

File: tunnel/cmd/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+`localproxy` is a Go implementation of [aws-samples/aws-iot-securetunneling-localproxy](https://github.com/aws-samples/aws-iot-securetunneling-localproxy).
+
+### Demo
+
+Following commands proxies connection to your local ssh server via `secure-tunnel-server`.
+
+1. Build the localproxy
+    ```shell
+    cd localproxy
```
Comment (kt3k) [inline tunnel/cmd/README.md:38]: Maybe `go build .` command is missing in the next line?

File: tunnel/cmd/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+3. Run the sourceproxy
+    ```shell
+    $ ./localproxy -access-token=${SOURCE_ACCESS_TOKEN} \
+        -source-listen-port=2222 \
+        -proxy-endpoint=localhost:443 \
+        -no-ssl-host-verify
+    ```
+4. Connect to the server via proxy
+    ```shell
+    $ ssh localhost -p 2222
```
Comment (kt3k) [inline tunnel/cmd/README.md:56]: I was able to login via localproxy on my machine! 👍

File: tunnel/cmd/README.md (hunk 1 above)
Comment (at-wat) [reply tunnel/cmd/README.md]: Thanks! Added the build command.

**Pull request merged**